flipperdevices/flipperzero-firmware
# [FL-1722] BLE custom serial service

# What's new

- Refactor device information service
- Add battery level service
- Add custom serial over BLE service. Implement "echo" server
- Advertise custom dummy services, depending on Flipper color

# Verification 

- Verify battery level notification
- Verify "echo" server by writing data to TX characteristic in Serial service and receiving the same data from RX characteristic

# Checklist (do not modify)
